diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5dc9bff5..ec27f7cb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -807,7 +807,7 @@ if(VRPN_BUILD_EXTRA_COMPILER_WARNINGS)
 	globally_enable_extra_compiler_warnings()
 endif()
 
-set(VRPN_CLIENT_ONLY)
+set(VRPN_CLIENT_ONLY OFF)
 if(VRPN_BUILD_CLIENT_LIBRARY AND NOT VRPN_BUILD_SERVER_LIBRARY)
 	# We can define VRPN_CLIENT_ONLY in the header in this case!
 	set(VRPN_CLIENT_ONLY ON)
@@ -1167,7 +1167,7 @@ if(VRPN_BUILD_CLIENT_LIBRARY)
 		PUBLIC_HEADER
 		${VRPN_CLIENT_PUBLIC_HEADERS})
 
-	if(NOT VRPN_CLIENT_ONLY)
+	if(VRPN_CLIENT_ONLY)
 		set_property(TARGET
 			vrpn
 			PROPERTY
diff --git a/vrpn_Connection.C b/vrpn_Connection.C
index dd986457..081f11f7 100644
--- a/vrpn_Connection.C
+++ b/vrpn_Connection.C
@@ -2423,7 +2423,7 @@ int vrpn_start_server(const char * machine, char * server_name, char * args,
 #if defined(sparc) || defined(FreeBSD) || defined(_AIX) || defined(__ANDROID__)
                     int status;  // doesn't exist on sparc_solaris or FreeBSD
 #else
-                    union wait status;
+                    int status;
 #endif
 		    
                     /* Check to see if they called back yet. */
